File: Assignment5/program2.c
///////////////////////////////////////////////////////////////////////////////////
//
//    File namev:  program2.c
//    Description: Accept the number from the user and print the number till given number 
//    Author :     Mayuresh Kaluskar
//    Date :       14/05/2025
//
////////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>

void Display(int iNo)
{
    int iCnt = 0 ;

    for(int iCnt = 1;iCnt<=iNo;iCnt++)
    {
        printf("%d\t",iCnt);
    }
}

int main()
{
    int iVal = 0;

    printf("Enter the number :");
    scanf("%d",&iVal);

    Display(iVal);

    return 0;

}

File: Assignment11/program2.java
import java.util.Scanner;

class DigitDisplay
{
    public void Display(int iNo)
    {
      
      for(int i = iNo;i>=1;i--)
      {
          System.out.print(i+"\t#\t");
      }
    }
}

class program2
{
    public static void main(String[] args) {
        int iVal = 0;

        Scanner sobj =new Scanner(System.in);

        System.out.println("Enter the number");
        iVal = sobj.nextInt();
         
        DigitDisplay dobj = new DigitDisplay();

        dobj.Display(iVal);
        
    }
}